src/pkg/backup/backup.go

```diff
@@ -70,7 +71,8 @@ type Backup struct {
 // interface compliance checks
 var _ print.Printable = &Backup{}
 
-func New(
+func CreateNewBackup(
```

Comment:
I'm lukewarm on this rename.  The package name is already `backup`, so there's a redundancy in calling `backup.CreateNewBackup`.  And `backup.CreateNew` is a longer version of `backup.Create` or `backup.New`.  Of the latter two, either one seems good to me.